deployment/README.md

```diff
@@ -40,13 +39,13 @@ bastion_ami           = "ami-08f3d892de259504d"
 bastion_instance_type = "t3.nano"
 bastion_ebs_optimized = true
 
-rds_database_identifier = floodmap-staging
+rds_database_identifier = floodmap
```

Comment:
This was a little finicky to rename. I wanted to preserve whatever data was in it.

My first thought was to snapshot the database, rename it in the console, and then re-import it into Terraform state, but I quickly ran into problems with attaching the database to subnets in the same VPC. Terraform got generally confused about how to associate existing resources with the renamed instance. I abandoned that approach pretty quickly so that I wouldn't create a lot of mess to manually clean up. I ended up:

1. taking a snapshot of the database
2. disabling deletion protection
3. destroying the database with Terraform
4. recreating the database from a snapshot
5. importing the new database into Terraform
6. running a plan and apply to bring all of the new RDS instance settings up to speed and create any dependent resources
7. manually removing the leftover database subnet group and parameter groups that got disassociated from the instance during my first approach

I expected the first approach to be a little thorny and had read up on some caveats around needing to update any resources interacting with that database's DNS name, but I was hopeful that Terraform would be able to figure it out better than it did.

Everything looks clean now, but I definitely learned some lessons about renaming RDS instances.

Reply:
This approach makes sense and is probably the safest. A potentially lighter weight alternative would have been to use PostgreSQL tooling to dump and restore the database contents.